powerdnsadmin/__init__.py

```diff
@@ -100,4 +100,9 @@ def inject_setting():
         setting = Setting()
         return dict(SETTING=setting)
 
+    @app.context_processor
+    def inject_mode():
+        setting = app.config['OFFLINE_MODE'] if 'OFFLINE_MODE' in app.config else 'False'
```

Comment:
We can replace this line with
```
setting = app.config.get('OFFLINE_MODE', False)
```

We should treat it as a boolean value instead of a "string" one. Otherwise, the variable you set in the docker-compose environment won't work.